test/lru.ml

```diff
@@ -7,7 +7,21 @@ module Test (K : sig
   val pp : t Fmt.t
 end) =
 struct
-  module Lru = Cachecache.Lru.Make (K)
+  module Cache = Cachecache.Lru.Make (K)
+
+  module DB = struct
+    type value = K.t
+    type key = K.t
+    type t = (K.t, value) Hashtbl.t
```

Comment:
I think you should use `Hashtbl.Make(K)` here instead of the polymorphic hashtbl

Reply:
I will try that change, isn't clearer like that, for types ?